assets/i18n/qqq.json

```diff
@@ -474,6 +474,8 @@
 	"I18N_SIGNUP_USERNAME_EXPLANATION": "Text displayed next to an entry box in the signup page - Explains how the username selected in the box is going to be used inside the site.",
 	"I18N_SIGNUP_WAIVER_OBJECTIVE": "Text displayed inside a information dialog widget in the signup page. - It explains the waiver of attribution. See I18N_SIGNUP_WHY_LICENSE.",
 	"I18N_SIGNUP_WHY_LICENSE": "Text displayed inside a information dialog widget in the signup page. - It's the title of a dialog that contains additional information about why the creative commons licence was chosen for Oppia.",
+	"I18N_SOLICIT_ANSWER_DETAILS_FEEDBACK": "Text for the feedabck after the student submits the answer details.",
+	"I18N_SOLICIT_ANSWER_DETAILS_QUESTION": "Text for the question which asks the students for answer details.",
```

Comment:
"... the students to explain why they entered a particular answer."

Reply:
Done